File: code/codes/js/write_comment.js
const ACCESS_TOKEN = "발급받은 ACCESS_TOKEN 입력";
const HOST = "https://openapi.band.us";
const API = "/v2/band/post/comment/create";

// parameters
const BAND_KEY = "검색할_밴드_키";
const POST_KEY = "검색할_포스트_키";
const BODY = "댓글 내용";

const xhr = new XMLHttpRequest();

xhr.onload = function () {
  if (xhr.status != 200) {
    // Error!
    return;
  }
  var responseObj = xhr.response;
  console.log(responseObj); // response!
};
xhr.open(
  "POST",
  `${HOST}${API}?access_token=${ACCESS_TOKEN}&band_key=${BAND_KEY}&post_key=${POST_KEY}&body=${BODY}`
);
xhr.setRequestHeader("Content-Type", "application/json");
xhr.responseType = "json";
xhr.send();

File: code/codes/ruby/write_comment.rb
require 'pp'
require_relative '0_env'

# https://developers.band.us/develop/guide/api/write_comment

# 댓글 작성
# 특정 글에 댓글을 작성합니다.

# コメント作成
# 特定のポストにコメントを作成。

params = {
    'band_key' => 'INPUT YOUR BAND KEY',
    'post_key' => 'INPUT YOUR POST KEY',
    'body' => "한글 日本語\nアイウエオ\n\nあいうえお\n\n\nEnglish"
}
response = post_request('https://openapi.band.us/v2/band/post/comment/create', params)
pp response

result_data = response['result_data']
properties = ['message']
puts validate?(result_data, properties)

message = result_data['message']
raise "NOT SUCCESS" unless message == 'success'